tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -294,10 +292,8 @@ def generate_batch(batch_size, num_skips, skip_window):
           top_k = 8  # number of nearest neighbors
           nearest = (-sim[i, :]).argsort()[1:top_k + 1]
           log_str = 'Nearest to %s:' % valid_word
-          for k in xrange(top_k):
-            close_word = reverse_dictionary[nearest[k]]
-            log_str = '%s %s,' % (log_str, close_word)
-          print(log_str)
+          
+          print(log_str, ', '.join(reverse_dictionary[nearest[k]] for k in range(top_k)))
```

Comment:
80 character line length max for Python code.

Reply:
I've already format the code to adapt for PEP8 style, please check. Thank you all.